funds-manager/funds-manager-server/src/custody_client/mod.rs

```diff
@@ -20,20 +27,24 @@ abigen!(
 );
 
 /// The source of a deposit
-pub(crate) enum DepositSource {
-    /// A Renegade quoter
+#[derive(Clone, Copy)]
+pub(crate) enum DepositWithdrawSource {
+    /// A Renegade quoter deposit or withdrawal
     Quoter,
-    /// A fee withdrawal
-    FeeWithdrawal,
+    /// A fee redemption deposit
+    FeeRedemption,
+    /// A gas withdrawal
+    RelayerGas,
```

Comment:
nit: we'll also need gas for the quoters (for permit2 approvals & deposits to custody) so we may wanna rename this

Reply:
good call